Add HTMLTableCellElement.{align|vAlign} (#32993)
mdn/content: files/en-us/web/api/htmltablecellelement/valign/index.md

---
title: "HTMLTableCellElement: vAlign property"
short-title: vAlign
slug: Web/API/HTMLTableCellElement/vAlign
page-type: web-api-instance-property
browser-compat: api.HTMLTableCellElement.vAlign
---

{{APIRef("HTML DOM")}}{{deprecated_header}}

The **`HTMLTableCellElement.vAlign`** property is a string indicating how to vertically align text in the cell.

> **Note:** This property is deprecated and CSS should be used to horizontally align text in a cell. Use the {{cssxref("vertical-align")}} property instead.

Reviewer: ```suggestion
> **Note:** This property is deprecated and CSS should be used to horizontally align text in a cell. Use the {{cssxref("vertical-align")}} property instead.
```

## Value

> The possible values are:

Author: Yes, you are right. Fixing. We don't need to list `justify` anymore, though.

- `top`
  - : Align the text to the top of the cell (default value). Use `vertical-align: top` instead.
- `middle`
  - : Vertically center the text in the cell. Use `vertical-align: middle` instead.
- `bottom`
  - : Align the text to the bottom of the cell. Use `vertical-align: bottom` instead.

> ### Examples

Reviewer: ```suggestion
## Examples
```

---
title: "HTMLTableCellElement: vAlign property"
short-title: vAlign
slug: Web/API/HTMLTableCellElement/vAlign
page-type: web-api-instance-property
browser-compat: api.HTMLTableCellElement.vAlign
---

{{APIRef("HTML DOM")}}{{deprecated_header}}

The **`vAlign`** property of the {{domxref("HTMLTableCellElement")}} interface is a string indicating how to vertically align text in the {{htmlelement("th")}} or {{htmlelement("td")}} table cell.

> > **Note:** This property is deprecated and CSS should be used to horizontally align text in a cell. Use the {{cssxref("vertical-align")}} property instead.

Reviewer 2: ```suggestion
> **Note:** This property is deprecated. Use the CSS {{cssxref("vertical-align")}} property to horizontally align text in a cell instead.
```

## Value

The possible values are: `"top"`, `"middle"`, `"bottom"`, or `"baseline"`

- `top`
  - : Align the text to the top of the cell. Use `vertical-align: top` instead.
- `center`
  - : Vertically center the text in the cell. Synonym of `middle`. Use `vertical-align: middle` instead.
- `middle`
  - : Vertically center the text in the cell. Use `vertical-align: middle` instead.
- `bottom`
  - : Align the text to the bottom of the cell. Use `vertical-align: bottom` instead.
- `baseline`
  - : Similar to `top`, but align the baseline of the text as close to the top so no part of the character is outside of the cell.

## Examples

> Use CSS `vertical-align` instead, which takes precedence. An [example](/en-US/docs/Web/CSS/vertical-align#vertical_alignment_in_a_table_cell) is available on the {{cssxref("vertical-align")}} page.

Reviewer 2: ```suggestion
Use CSS {{cssxref("vertical-align")}} instead, which takes precedence, as demonstrated in the [vertical alignment table cells](/en-US/docs/Web/CSS/vertical-align#vertical_alignment_in_a_table_cell) example.
```
better to link descriptive text.
